make.inc

```diff
@@ -54,7 +54,7 @@ CFLAGS   += -std=c99
 # libraries
 
 # gcc with OpenBLAS (includes LAPACK)
-LIB       = -lopenblas
+LIB       = -lopenblas64_
```

Comment:
After that changing in the suffix, ld still could not find our functions with _64_.
So I supposed that the renaming of the lib of OpenBLAS caused the problem.
Therefore, I edited the make.inc file to see if it helps.